source/Renderer/Renderer.LoadingScreen.cs

```diff
@@ -127,23 +126,23 @@ internal static void DrawLoadingScreen()
 				if (TextureLogo != null)
 				{
 					// place the centre of the logo at from the screen top
-					int logoTop = (int) (Screen.Height*logoCentreYFactor - TextureLogo.Height/2.0);
+					int logoTop = (int) (ScreenHeight*logoCentreYFactor - TextureLogo.Height/2.0);
 					logoBottom = logoTop + TextureLogo.Height;
-					LibRender.Renderer.DrawRectangle(TextureLogo, new Point((Screen.Width - TextureLogo.Width)/2, logoTop),new Size(TextureLogo.Width, TextureLogo.Height), Color128.White);
+					DrawRectangle(TextureLogo, new Point((int)(ScreenWidth - TextureLogo.Width)/2, logoTop),new Size(TextureLogo.Width, TextureLogo.Height), Color128.White);
 				}
 			}
 			else
 			{
 					// if custom route image, no logo and leave a conventional black area below the potential logo
 			}
-				logoBottom	= Screen.Height / 2;
+				logoBottom	= (int)ScreenHeight / 2;
 			// take the height remaining below the logo and divide in 3 horiz. parts
-			int	blankHeight	= (Screen.Height - logoBottom) / 3;
+			int	blankHeight	= (int)(ScreenHeight - logoBottom) / 3;
 
 			// VERSION NUMBER
 			// place the version above the first division
 			int	versionTop	= logoBottom + blankHeight - fontHeight;
-			LibRender.Renderer.DrawString(Fonts.SmallFont, "Version " + typeof(Renderer).Assembly.GetName().Version,
+			DrawString(Fonts.SmallFont, "Version " + typeof(Renderer).Assembly.GetName().Version,
```

Comment:
Need to either pass in the version to render or set the assembly version for the renderer too